flexmeasures/app.py

```diff
@@ -106,7 +106,7 @@ def create(  # noqa C901
     set_secret_key(app)
     if app.config.get("SECURITY_PASSWORD_SALT", None) is None:
         app.config["SECURITY_PASSWORD_SALT"] = app.config["SECRET_KEY"]
-    if app.config.get("FLEXMEASURES_ENV") not in ("documentation", "development"):
+    if app.config.get("FLEXMEASURES_FORCE_HTTPS", False):
```

Comment:
This could be a "breaking" for users as they could be expecting the `SSLify` to happen given the value of `FLEXMEASURES_ENV`. Should we set `FLEXMEASURES_FORCE_HTTPS` to True in such case or rather inform users in the release. I favor the latter option.

Reply:
I agree. I'll add a warning in the release notes.